integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,49 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            members: '[.members[].user.email]'
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .team_id + \"/v/l/li/\" + .id"
+            startDate: .start_date | tonumber / 1000 | todate
+            endDate: .due_date | tonumber / 1000 | todate
+            totalIssues: .task_count
+          relations:
+            team: .__team_id
+  - kind: issue
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupIssue"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            url: .url
+            status: .status.status
+            assignee: .assignees | map(.email) | first
```

Comment:
why not all assignees? having all the list will allow seeing the distribution and spread work inside port between the different assignees, while only setting the first, might be misleading as it can be ordered alphabetically

Reply:
It is a pre-defined blueprint and was advised to not modify. I can adjust it to take in all assignees